test/functional/helpers/report-helper.js

```diff
@@ -0,0 +1,63 @@
+function checkTestRelevanceByBrowserAlias (test, browserAlias) {

```

Comment:
The module exports just one function. So, it should be called by this function name. (For example, if the module exports just a function that reads any file, the module should be called `read-file.js`). Discuss with a tech writer please